client/src/components/ImageRotateButton.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const ImageRotateButton = ({ direction, disable, handleRotate }) => (
+  <div
+    className={handleRotate ? 'image-carousel-show' : 'image-carousel-disable'}
+    id={direction > 0 ? 'image-carousel-next' : 'image-carousel-prev'}
+    onClick={disable ? () => {} : () => { handleRotate(direction); }}
+    onKeyUp={() => {}} // this and the lower 2 lines are solely for eslint
```

Comment:
Comment for eslint.  Might look unprofessional to potential employers.  But it also might be helpful to you in the project.  Up to you.